js/sparkswap.js

```diff
@@ -271,7 +271,46 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
-        return this.privateGetV1OrderId ({ id });
+        const order = await this.privateGetV1OrdersId ({ id });
```

Comment:
I thought destructuring didn't work?

Reply:
Oh it totally does, don't know why I thought that.

Reply:
@martineehrlich you are right. destructuring doesn't work for ccxt, this is an example composition.

if it was `const { order } = await....` then we would get errors from transpiling